share/fathead/plural/extract_plurals.pl

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/perl
+
+use Modern::Perl 2013;
+use autodie;
+use XML::Twig;
+use URI::Escape qw(uri_escape_utf8);
+use Lingua::EN::Inflect qw(WORDLIST);
+use utf8;
+$|++;
+
+#Regex to catch weirdness (malformed tags etc.)
+# we don't want leaking into the results
+my $unsanitary = qr/[^\p{L}\s\-']/;
+
+#Define triggers
+my @triggersStart = ("plural of", "pluralise", "pluralize", "what is the plural of");
```

Comment:
You can skip these and just have one entry per word.  I'll add these as trigger words on the backend.